### PR TITLE
Convert to toml

### PR DESCRIPTION
**Pull request by aquifolly**
the latest stable version of halloy uses toml instead of yaml

https://github.com/squidowl/halloy/releases/tag/2024.5

### CODE REVIEW

**Review by isabelroses — COMMENTED** [review]
You might also want to update the readme

**Comment by aquifolly** [comment]
ah yes

**Comment by uncenter** [comment]
Once we get this merged LMK and I can convert this over to Whiskers :)

**Comment by Opposite34** [comment]
@pinkcreeper100 you might want to change a line in the README:
`theme: "catppuccin-frappe"` to `theme = "catppuccin-frappe"` (with an equal sign)

File: README.md [hunk 1]
```diff
@@ -37,11 +37,11 @@
 ## Usage
 
 1. Clone this repository locally
-2. Copy the desired `<theme>.yaml` file into your themes directory:  
+2. Copy the desired `<theme>.toml` file into your themes directory:  
     - Linux: `$HOME/.config/halloy/themes`
     - macOS: `$HOME/Library/Application Support/halloy/themes`
     - Windows: `{FOLDERID_RoamingAppData}\halloy\themes`
-4. Update the `theme:` field of your `config.yaml` to the name of the theme without the `.yaml` file extension:
+4. Update the `theme:` field of your `config.toml` to the name of the theme without the `.toml` file extension:
```
Comment (uncenter) [inline README.md]:
```suggestion
4. Update the `theme = "..."` field of your `config.toml` to "catppuccin-<flavor>".
```

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
Any chance you can take a look at this @Daeraxa?

**Comment by Daeraxa** [comment]
Apologies, I've been away from open source things for a little while. Rather than keep this hanging around any longer I'm happy to go with it if people have tested it on a current version of Halloy and can confirm it. Otherwise I'll update my version, check the themes out and, if necessary, re-create the screenshots if anything major has changed in the UI but might take me a little bit to get to as I've got a bunch of other projects I've also been neglecting.

**Comment by sgoudham** [comment]
We'll want to merge #2 instead of this one as it combines the effort of migrating to whiskers and creating the TOML files so I'm going to close this as not planned. 

Sorry @pinkcreeper100!

**Comment by aquifolly** [comment]
perfectly fine with me

**Pull request closed without merging**